### DISCUSSION TITLE
Preview issues - new card links do not work; some images do not display

### DISCUSSION THREAD

**Discussion by JaniceManwiller** [GitbookIO/community#278] [0. General bugs and questions]
I am previewing a large CR that includes several new topics. I also added new cards to existing card sets and a new card set.

In the preview:

- Several images are marked as "cannot display". The images display fine in the CR itself. Tried replacing one of the images completely (deleted the image object and added a new one), but it still wouldn't display.
- All links from new cards, whether in an existing card set or a new card set, do not work. I tried resetting the new card link and then reloading the preview, but the links still do not work.

I also tried restarting the browser, but that did not help.

I really need to publish these updates, but do not want to publish broken images and links. I'm not sure if the issue is just with the preview. Everything seems fine in the CR.

**Comment by JaniceManwiller** [comment]
I also tried a different browser, but had the same issues.

**Reply by addisonschultz** [reply]
Hi @JaniceManwiller - thanks for flagging! This is a weird bug that just popped up, but we're already almost done fixing it. We should have the fix shipped soon. Sorry for the inconvenience!

**Comment by valentin0h** [comment]
Thank you for reporting this @JaniceManwiller. We are aware of the issue and will ship a fix very soon.

**Comment by JaniceManwiller** [comment]
Is the issue just with Preview? Or will the same issues be in the published doc?

**Reply by valentin0h** [reply]
The issue is only with the _Preview_ of the change request.

**Comment by dfinster** [comment]
Is the ETA measured in hours or days? 
We have a large change under review by a committee with these broken images and the reviewers cannot approve publication with the broken preview. Assuring them that "don't worry, it's only in the preview", or clicking and hoping it works isn't going to be possible.

**Comment by addisonschultz** [comment]
Hi @JaniceManwiller @dfinster — This issue should be fixed, I've just tested and verified on my end! Thanks for flagging, and please let us know if it's happening again

**Reply by dfinster** [reply]
@addisonschultz We are seeing this issue again this morning. I added new images to cards today, and they are not visible in Preview. This is interfering with a high-priority review cycle today.

**Reply by addisonschultz** [reply]
Thanks for flagging - I've been testing on my end with new and existing spaces, but not able to reproduce this again. 

Are you using images that were uploaded before this was an issue? It shouldn't matter, but trying to debug a bit more - the images you're adding are brand new uploaded ones? Can you let me know what format images they are as well?

**Reply by dfinster** [reply]
The instance I found this morning is a new image added to an existing card. The image is PNG format, I created it in GIMP. If it matters, this is a large change request that is more than two months old and has more than 3100 edits. I hope it's approved soon so we can merge it, they seldom get this old or big.

**Reply by addisonschultz** [reply]
Are the cards you're working with the only ones that are still giving you issues? Or is the original issue (Page links, and normal images [not card images]) still happening as well?

**Comment by dfinster** [comment]
@addisonschultz 
At present I only see the issue in a single card. This is an existing 2x2 card layout that I replaced the lower-right image this morning with a new one. For illustration, here are screenshots of the edit page, and the preview page. Apologies for the blur, these are drafts that haven't been approved for publication yet.

* Edit page:
![image](https://github.com/GitbookIO/community/assets/106555692/55952e43-6210-4e82-85cf-5bc572ae859f)

* Preview page:
![image](https://github.com/GitbookIO/community/assets/106555692/a7e6ebaf-9565-4711-831e-e98e960e7633)

**Reply by addisonschultz** [reply]
Hi @dfinster - are you able to share the preview link at all? I'd like to debug to see what's going wrong. You're able to reach me at:

addison at gitbook.com
